src/components/funfacts/funfact-slideshow/index.tsx

```diff
@@ -1,3 +1,2 @@
 import FunfactSlideshow from './FunfactSlideshow';
-
 export default FunfactSlideshow;
```

Comment:
here you can write `export {default} from './FunfactsSlideshow`. No biggie, but nice to know